test_bst.py

```diff
@@ -0,0 +1,63 @@
+from bst import BinarySearchTree
+
+
+nodes = [5, 4, 8, 3, 43, 22, 7, 74, 2]
+
+
+def test_insert_nodes():
+    """size of tree equal to number of values inserted"""
+    b = BinarySearchTree()
+    for n in nodes:
+        b.insert(n)
+    assert b.size() == 9
```

Comment:
You might ensure that the values you've inserted are in the tree.  You might test what happens when you insert a duplicate value.